apps/docs/pages/guides/platform/sso/sso-with-azure.mdx

```diff
@@ -0,0 +1,127 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+import { Button, IconArrowDown } from 'ui'
+
+export const meta = {
+  id: 'sso-with-azure',
+  title: 'Setup SSO with Azure AD',
+  description: 'Configuration instructions for single sign-on with Azure AD (Entra)',
+}
+
+Supabase supports single sign-on (SSO) using Microsoft Azure AD.
+
+## Step-by-step guide
+
+### Step 1: Add and register an Enterprise Application
```

Comment:
Adding custom anchors to make the URL a bit cleaner, easier to link to, and maintain in the future.
```suggestion
## Step 1: Add and register an Enterprise Application [#add-and-register-enterprise-application]
```